calico_node/Makefile

```diff
@@ -428,10 +428,9 @@ semaphore:
 	$(MAKE) node-test-containerized
 
 	# Actually run the tests (refreshing the images as required), we only run a
-	# small subset of the tests for testing SSL support.  These tests are run
-	# using "latest" tagged images.
-	$(MAKE) $(NODE_CONTAINER_NAME) st
-	ST_TO_RUN=tests/st/policy $(MAKE) st-ssl
+	# small subset of the tests for testing SSL support.
+	FELIX_VER=master $(MAKE) $(NODE_CONTAINER_NAME)
```

Comment:
I see that, as it's currently laid out, that can't easily be done since the VER variables are calculated up-front.